types/materialize-css/index.d.ts

```diff
@@ -77,7 +77,199 @@ declare global {
+             * If the picker is open.
+             */
+            isOpen: boolean;
+
+            /**
+             * The selected Date.
+             */
+            date: Date;
+
+            /**
+             * Open datepicker
+             */
+            open(): void;
+
+            /**
+             * Close datepicker
+             */
+            close(): void;
+
+            /**
+             * Gets a string representation of the selected date
+             */
+            toString(): string;
+
+            /**
+             * Set a date on the datepicker
+             * @param date Date to set on the datepicker.
+             */
+            setDate(date?: Date): void;
+
+            /**
+             * Change date view to a specific date on the datepicker
+             * @param date Date to show on the datepicker.
+             */
+            gotoDate(date: Date): void;
+        }
+
+        interface DatePickerOptions {
+            /**
+             * The date output format for the input field value.
+             * @default 'mmm dd, yyyy'
+             */
+            format: string;
+
+            /**
+             * Used to create date object from current input string.
+             */
+            parse: (value: string, format: string) => Date;
+
+            /**
+             * The initial date to view when first opened.
+             */
+            defaultDate: Date;
+
+            /**
+             * Make the `defaultDate` the initial selected value
+             * @default false
+             */
+            setDefaultDate: boolean;
+
+            /**
+             * Prevent selection of any date on the weekend.
+             * @default false
+             */
+            disableWeekends: boolean;
+
+            /**
+             * Custom function to disable certain days.
+             */
+            disableDayFn: (day: Date) => boolean;
+
+            /**
+             * First day of week (0: Sunday, 1: Monday etc).
+             * @default 0
+             */
+            firstDay: number;
+
+            /**
+             * The earliest date that can be selected.
+             */
+            minDate: Date;
+
+            /**
+             * The latest date that can be selected.
+             */
+            maxDate: Date;
+
+            /**
+             * Number of years either side, or array of upper/lower range.
+             * @default 10
+             */
+            yearRange: number | number[];
+
+            /**
+             * Changes Datepicker to RTL.
+             * @default false
+             */
+            isRTL: boolean;
+
+            /**
+             * Show month after year in Datepicker title.
+             * @default false
+             */
+            showMonthAfterYear: boolean;
+
+            /**
+             * Render days of the calendar grid that fall in the next or previous month.
+             * @default false
+             */
+            showDaysInNextAndPreviousMonths: false;
```

Comment:
You want `boolean` - not `false`, right?

Reply:
Thanks, fixed that.